From bee0e8e091b70f6235309bb2ec3ee5a93e7390f7 Mon Sep 17 00:00:00 2001
From: Brett Brotherton <bbrotherton@google.com>
Date: Fri, 19 Jul 2024 09:06:52 -0600
Subject: [PATCH] [PATCH] select non libz Crc32 implementation

use the older builtin Crc32 implementation rather than importing the
zlib third party library for this single function

Change-Id: I7fc55eb548130e28b78c54fcb030c15e61192553
---
 base/metrics/crc32.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/base/metrics/crc32.cc b/base/metrics/crc32.cc
index 83e3cee257..1e24bd79e6 100644
--- a/base/metrics/crc32.cc
+++ b/base/metrics/crc32.cc
@@ -4,13 +4,13 @@
 
 #include "base/metrics/crc32.h"
 
-#if !BUILDFLAG(IS_NACL)
+#if !BUILDFLAG(IS_NACL) && 0
 #include "third_party/zlib/zlib.h"
 #endif  // !BUILDFLAG(IS_NACL)
 
 namespace base {
 
-#if !BUILDFLAG(IS_NACL)
+#if !BUILDFLAG(IS_NACL) && 0
 uint32_t Crc32(uint32_t sum, span<const uint8_t> data) {
   if (data.empty()) {
     return sum;
-- 
2.45.2.1089.g2a221341d9-goog

